UnitsNet.NanoFramework/GeneratedCode/AmountOfSubstance/packages.config

```diff
@@ -1,4 +1,4 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="nanoFramework.CoreLibrary" version="1.10.4-preview.11" targetFramework="netnanoframework10" />
```

Comment:
I don't think this PR should have to touch these nanoframework files? 
Make sure you have merged in latest origin/master and then maybe revert these.

Reply:
> That is my question as well- we can easily have both the Generic & Extended schemas implemented- but should we and which one would you recommend?

I don't like the extended schema. If some quantities need quoted numbers, then I think it would be simpler to just quote all numbers.

I have to admit I am struggling to grasp all the contracts, and surrogates and converters here and in #966.
That table alone is mind-boggling to put it mildly 😅 so bear with me if I am not quite getting it.

How about these two contracts:
- **Shape A**: Quoted number + full enum name
- **Shape B**: Quoted number + unit abbreviation

It could then look like this:
- WCF and its `DataContractSerializer` (xml) or `DataContractJsonSerializer` (json)
  - Shape A: `GenericQuantityDataContractSurrogate`
  - Shape B: `QuantityWithAbbreviationContractSurrogate`
- Json.NET and its `JsonConverter`
  - Shape A: `GenericQuantityJsonConverter` (but with quoted numbers)
  - Shape B: `QuantityWithAbbreviationJsonConverter`
- System.Text.Json and its `JsonSerializer`
  - Shape A: #966 ? 
  - Shape B: #966 ?
- Protobuf and other data contract compatible serializers
  - I assume similar converters as above must be created for each? That pesky decimal seems to be the main problem.

Reply:
Those got auto-incremented on the last commit (after running the build script)- I didn't follow much of the discussion around the nano-framework, so I thought this was intentional.

Reply:
Yes, sorry, that was a flaw in the setup earlier. It should have been fixed by now.

Reply:
I've updated the summary: 

1. From what I gather there should be no backwards support for the Basic & Extended schemas (will remove the corresponding Surrogates). 
2. All values should be quoted: will update the contracts & tests
3. This would break the compatibility tests- so those should be removed as well 
4. Should I add the new JsonNet converters? System.Text?
5. If (4) then should I keep the DC <-> JsonNet compatibility tests? Maybe change them something about them? I first wanted to make it work with strings and quantities- but the format is never exactly the same : there is a difference in serializing slashes- like in "mg/l", so I went with the serialize-with-one -deserialize-with-another workflow..
6. Nano packages should be reverted

Reply:
1. I think breaking backwards compatibility for new converters and new contracts is fine.
2. Yes
3. Yes
4. json.net and system.text converters can be added in separate PRs, if it makes sense to scope this PR can to only the data contracts. You decide. Also I don't know PR 966 well enough, if you should reuse that work or not.
5. If I understand correctly, Shape A and Shape B should have compatible JSON formats regardless of DC, JsonNet or SystemText being used. If so, then yes, compatibility tests should be kept.
6. Yes